packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,23 +1,10 @@
-## 2.0.0-nullsafety.4
+## 2.0.0
 
+* **Breaking changes**:
```

Comment:
We haven't been doing this in general, since it's sort of obvious 🤷

Reply:
Hah you're right, let me remove drama from here. In other plugins with more changes it makes more sense, for example the file_selector had 2 breaking changes :P